internal/server/authn/errors.go

```diff
@@ -0,0 +1,9 @@
+package authn
+
+import "fmt"
+
+var (
+	ErrInvalidProviderURL          = fmt.Errorf("invalid provider url")
+	ErrInvalidProviderClientID     = fmt.Errorf("invalid client id")
+	ErrInvalidProviderClientSecret = fmt.Errorf("invalid client secret")
```

Comment:
do we want to inherit from errors that we know how to render to the user?

Reply:
Good call-out, made this change:
https://github.com/infrahq/infra/pull/1745/commits/6c9c255725cabde8cc6436b4e0678bd26d0bea91